### PR TITLE
Two small improvements

### PR DESCRIPTION
**Pull request by famz**

### CODE REVIEW

**Comment by stefanha** [comment]
Thanks, I have merged the first commit.  Please see the comment on the second commit for why it cannot be merged.

**Comment by famz** [comment]
Thanks, but I don't see a comment?

**Comment by stefanha** [comment]
Sorry, didn't realize I need to "Submit Review" in order for the comment to show up.

File: git-publish [hunk 1]
```diff
@@ -551,7 +551,7 @@ def main():
                 print 'Using defaults when a non-default profile exists. Forgot to pass --profile ?'
         else:
             print 'Profile "%s" does not exist, please check .gitpublish or git-config(1) files' % options.profile_name
-            return 1
+        return 1
```
Comment (famz) [inline git-publish]: Does 2 mean that "default" is always available (as an empty config)?

**Comment by stefanha** [comment]
On Tue, Aug 22, 2017 at 6:50 AM, Fam Zheng <notifications@github.com> wrote:

> *@famz* commented on this pull request.
> ------------------------------
>
> In git-publish
> <https://github.com/stefanha/git-publish/pull/26#discussion_r134389629>:
>
> > @@ -551,7 +551,7 @@ def main():
>                  print 'Using defaults when a non-default profile exists. Forgot to pass --profile ?'
>          else:
>              print 'Profile "%s" does not exist, please check .gitpublish or git-config(1) files' % options.profile_name
> -            return 1
> +        return 1
>
> Does 2 mean that "default" is always available (as an empty config)?
>

Either "default" can be defined in the .gitpublish file or it is treated as
an empty config.

Stefan

**Comment by famz** [comment]
On Mon, 08/21 22:55, Stefan Hajnoczi wrote:
> On Tue, Aug 22, 2017 at 6:50 AM, Fam Zheng <notifications@github.com> wrote:
> 
> > *@famz* commented on this pull request.
> > ------------------------------
> >
> > In git-publish
> > <https://github.com/stefanha/git-publish/pull/26#discussion_r134389629>:
> >
> > > @@ -551,7 +551,7 @@ def main():
> >                  print 'Using defaults when a non-default profile exists. Forgot to pass --profile ?'
> >          else:
> >              print 'Profile "%s" does not exist, please check .gitpublish or git-config(1) files' % options.profile_name
> > -            return 1
> > +        return 1
> >
> > Does 2 mean that "default" is always available (as an empty config)?
> >
> 
> Either "default" can be defined in the .gitpublish file or it is treated as
> an empty config.
> 

OK, then the current code makes sense. Thanks.

**Pull request closed without merging**